platform/pulp/platform/models/consumer.py

```diff
@@ -25,14 +25,14 @@ class Consumer(Model):
     :cvar notes: Arbitrary information about the consumer.
     :type notes: fields.GenericRelation
 
-    :cvar distributors: Associated distributors.
-    :type distributors: models.ManyToManyField
+    :cvar publishers: Associated publishers.
+    :type publishers: models.ManyToManyField
     """
     name = models.TextField(db_index=True, unique=True)
     description = models.TextField(blank=True)
 
     notes = fields.GenericRelation(Notes)
-    distributors = models.ManyToManyField('RepositoryDistributor')
+    publishers = models.ManyToManyField('Publisher')
```

Comment:
Not sure if this has been discussed elsewhere, but django's auto-naming of the relationship on the other model is awkward. I think it would appear in this case as ``consumer_set``. It's common to use the ``related_name`` parameter for the ``ManyToManyField`` and ``ForeignKey`` to provide a more natural name, such as ``consumers``. I suggest we do that here and anywhere else we have similar relationships.